File: 原創or不分類題庫/Python/a240.py
from sys import stdin

for m in stdin:
    times = int(m)

    for i in range(times):
        for n in stdin:
            digit = 1
            s = 0
            current = 0

            for j in range(int(n)):
                digit *= 10
                current = digit // 17
                s += current
                digit %= 17
            
            print(current, s, sep=" ")

File: 原創or不分類題庫/CPP/a240.cpp
#include <iostream>

using namespace std ;

int main () {

	int m, n ;

	while ( cin >> m ) {

		int digit, current, sum ;

		for ( int i = 0 ; i < m ; i ++ ) {
			cin >> n ;
			digit = 1 ;
			sum = 0 ;

			for ( int j = 0 ; j < n ; j ++ ) {
                digit *= 10 ; current = digit / 17 ;
				sum += current ;
				digit = digit % 17 ;
			}

			cout << current << " " ;
			cout << sum << endl ;
		}
	}

	return 0 ;

}
